stubs/Pygments/pygments/__init__.pyi

```diff
@@ -0,0 +1,5 @@
+from typing import Any, Optional
```

Comment:
Why not PEP 604 instead of `Optional`?

Reply:
Stubgen created a bunch of `Optional`s and I didn't change all of them manually. Should I?

Reply:
Don't worry. For future reference: stubgen from mypy 0.910+ uses PEP 604 syntax (and has a few other related benefits).